Makefile

```diff
@@ -0,0 +1,40 @@
+CONFIGURATION   := Debug
+NUNIT_CONSOLE   := packages/NUnit.ConsoleRunner.3.2.1/tools/nunit3-console.exe
+OS              := $(shell uname)
+RUNTIME         := mono --debug=casts
+V               ?= 0
+
+include build-tools/scripts/msbuild.mk
+
+all:
+	$(MSBUILD) $(MSBUILD_FLAGS) Xamarin.Android.Tools.sln
+
+clean:
+	-$(MSBUILD) $(MSBUILD_FLAGS) /t:Clean Xamarin.Android.Tools.sln
+
+prepare:
+	nuget restore Xamarin.Android.Tools.sln
+
+run-all-tests: run-nunit-tests
+
+# $(call RUN_NUNIT_TEST,filename,log-lref?)
+define RUN_NUNIT_TEST
+	MONO_TRACE_LISTENER=Console.Out \
+	$(RUNTIME) --runtime=v4.0.0 \
```

Comment:
should this not just be `v4.0` ? Not sure what windows will do with `v4.0.0`

Reply:
It's different/worse from that: `--runtime` is a _mono_ option, and if mono is not used, there will thus be no command _at all_, which is bad.

On the ¯\_(ツ)_/¯ side, this is what xamarin-android does...

I'll remove the `--runtime` option. It might not be needed anymore now that I'm using NUnit 3.2.1.

Reply:
I think the newer NUnit does a better job at detecting mono.

Reply:
I tend to use the --framework option though..